packages/api/README.md

```diff
@@ -0,0 +1,54 @@
+# @formbricks/api - API Wrapper for Formbricks
+
+## Installation
+
+```bash
+npm install @formbricks/api
+```
+
+## Usage
+
+<details>
+<summary>Create API Client</summary>
+
+```ts
+import { FormbricksAPI, EnvironmentId } from "@formbricks/api";
+
+const api = new FormbricksAPI({
+  apiHost: "http://localhost:3000",
+  environmentId: "clgwh8maj0005n2f66pwzev3r" as EnvironmentId,
+});
+```
+
+</details>
+
+NOTE: All of the following methods return a `Result` from the `@formbricks/errors` package.
+
+<details>
+<summary>Create a new response</summary>
+
+```ts
+const response = await api.createResponse({
+  surveyId: "......" as SurveyId,
+  personId: "......" as PersonId,
+  data: {
+    questionId: "response",
```

Comment:
The developer shouldn't type the values manually, they are going to be returned from a function (doesn't exist yet) so I don't think it would be a good idea to put example values there?